requirements.txt

```diff
@@ -10,3 +10,7 @@ cairocffi
 Sphinx
 python-decouple==2.3
 fig==1.0.1
+CairoSVG==1.0.13
+cffi==0.9.2
+pycparser==2.10
+selenium==2.45.0
```

Comment:
The requirements file already includes a reference to 'selenium'.  Removing the versionless reference made the requirements install with pip.